api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -332,6 +332,7 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${CATALOG_CODE} ${JAVA_BIN_DIR}java \
     -Dloader.path=${COMMON_LIB} \
     -Djava.library.path=${LIBPATH} \
     -Djavax.net.debug=${ZWE_configs_sslDebug:-""} \
+    -Dotel.sdk.disabled=true \
```

Comment:
Remove it at all or allow configure in advance
```suggestion
    -Dotel.sdk.disabled=${ZWE_component_config_otel_sdk_disabled:-false} \
```

Reply:
At this point, we do not want anyone to enable otel for microservices.